FRBDK/Glue/Glue/Controls/MainPanelControl.xaml

```diff
@@ -5,23 +5,132 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
              xmlns:controls="clr-namespace:FlatRedBall.Glue.Controls"
              xmlns:viewModels="clr-namespace:GlueFormsCore.ViewModels"
+             xmlns:plugins="clr-namespace:FlatRedBall.Glue.Plugins"
              mc:Ignorable="d" 
              d:DesignHeight="450" d:DesignWidth="800"
              PreviewKeyDown="UserControl_PreviewKeyDown"
              MouseDown="UserControl_MouseDown"
-             d:DataContext="{d:DesignInstance Type=viewModels:TabControlViewModel, IsDesignTimeCreatable=True}"
+             d:DataContext="{d:DesignInstance Type=viewModels:TabControlViewModel, IsDesignTimeCreatable=False}"
              >
     <UserControl.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <!--<ResourceDictionary Source="/Themes/Dark.xaml" />-->
                 <ResourceDictionary Source="/Themes/Light.xaml" />
             </ResourceDictionary.MergedDictionaries>
-        </ResourceDictionary>
-    </UserControl.Resources>
 
+            <DataTemplate x:Key="TabItemTemplate" DataType="{x:Type plugins:PluginTab}">
```

Comment:
These verbose resources are here somewhat temporarily. They are styled to look as close as possible to the current style in FRB, but use viewmodel bindings rather than what was originally implemented in code. This pries things open for cleaner control and allow more consistency to how things will be styled. They may possibly move to a separate file, but at the very least will be revisited for theming/styling.